pkg/asset/agent/ignition.go

```diff
@@ -0,0 +1,67 @@
+package agent
+
+import (
+	"path/filepath"
+
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
+	"github.com/openshift/installer/pkg/asset"
+	am "github.com/openshift/installer/pkg/asset/agent/manifests"
+	"github.com/openshift/installer/pkg/asset/ignition"
+	"github.com/openshift/installer/pkg/asset/ignition/bootstrap"
+)
+
+const manifestPathInIso = "/etc/assisted/manifests"
+
+// Ignition is an asset that generates the agent installer ignition file.
+type Ignition struct {
+	bootstrap.Common
```

Comment:
If we're going to use this type we should do it via composition rather than embedding. We don't want to inherit any of the implementation from it - as it is, this makes `Ignition` a `WritableAsset` with an implementation of `Files()` that we don't want.